### PR TITLE
[Issue #565] Add more information about inspecting the original certificate 

### PR DESCRIPTION
**Pull request by puglieri**

### CODE REVIEW

File: docs/general/https-filtering/known-issues.md [hunk 1]
```diff
@@ -30,7 +30,38 @@ All the issues known to us and the ETAs on their fixes are listed below.
...
 
-This problem is solved in [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). This browser extension helps you manage filtering directly from the browser and allows you to inspect the original certificate of any website.
+This problem can be solved in 2 ways:
+
+- By using our [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). This browser extension helps you manage filtering directly from the browser and allows you to inspect the original certificate of any website
+
+![Certificate Browser Assistant](https://cdn.adtidy.org/content/kb/ad_blocker/general/cert-browser.png)
+
+- By visiting the section *Recent activity* on AdGuard for Android or *Filtering log* on AdGuard for Windows and AdGuard for Mac. While in the Browser Assistant you can only see the certificate of the website, in Recent activity you can inspect the certificate of any server used in a subrequest or browser, with or without the Browser Assistant. To view the certificate using this method, follow the instructions below.
```
Comment (el-termikael) [inline docs/general/https-filtering/known-issues.md:39]:
```suggestion
- By visiting the *Recent activity* section in AdGuard for Android or the *Filtering log* section in AdGuard for Windows and AdGuard for Mac. While in the Browser Assistant you can only see the certificate of the website, in Recent activity you can inspect the certificate of any server used in a subrequest or browser, with or without the Browser Assistant. To view the certificate using this method, follow the instructions below.

File: docs/general/https-filtering/known-issues.md [hunk 2]
```diff
@@ -30,7 +30,38 @@ All the issues known to us and the ETAs on their fixes are listed below.
...
+
+On AdGuard for Android:
```
Comment (el-termikael) [inline docs/general/https-filtering/known-issues.md:41]: In — here and below

File: docs/general/https-filtering/known-issues.md [hunk 3]
```diff
@@ -30,7 +30,38 @@ All the issues known to us and the ETAs on their fixes are listed below.
...
+3. Click the request whose certificate you want to inspect to open the request details.
+4. Scroll down to *Original certificate*. Click for more info.
+
+![Recent activity AdGuard for Android](https://cdn.adtidy.org/content/kb/ad_blocker/general/cert-android.png)
+
+On AdGuard for Windows:
+
+1. Go to *Settings* → *Ad Blocker* → *Filtering log*.
+2. Click the request whose certificate you want to inspect to open the request details.
+3. Under the section *TLS*, click *View website certificate*.
```
Comment (el-termikael) [inline docs/general/https-filtering/known-issues.md:54]:
```suggestion
3. Under the *TLS* section, click *View website certificate*.

File: docs/general/https-filtering/known-issues.md [hunk 4]
```diff
@@ -30,7 +30,38 @@ All the issues known to us and the ETAs on their fixes are listed below.
...
+
+![Filtering log AdGuard for Windows](https://cdn.adtidy.org/content/kb/ad_blocker/general/cert-win.png)
+
+On AdGuard for Mac:
+
+1. Click the gear icon. On the dropdown menu, go to *Advanced* → *Filtering log...*.
```
Comment (el-termikael) [inline docs/general/https-filtering/known-issues.md:60]:
```suggestion
1. Click the gear icon. In the drop-down menu, go to *Advanced* → *Filtering log...*.

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/general/https-filtering/known-issues.md [hunk 5]
```diff
@@ -30,7 +30,38 @@ All the issues known to us and the ETAs on their fixes are listed below.
 
 The most important drawback of the HTTPS filtering mechanism is that it hides the real certificate of a website. You cannot simply check its original certificate because you can only see the one issued by AdGuard.
 
-This problem is solved in [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). This browser extension helps you manage filtering directly from the browser and allows you to inspect the original certificate of any website.
+This problem can be solved in 2 ways:
+
+- By using our [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). This browser extension helps you manage filtering directly from the browser and allows you to inspect the original certificate of any website
```
[the remark this answers is not in the document]
Comment (ameshkov) [reply docs/general/https-filtering/known-issues.md:37]: No change in the formatting yet (the image is a nested item here).

**Review by ameshkov — CHANGES_REQUESTED** [review]

**Comment by ameshkov** [comment]
Other [than this](https://github.com/AdguardTeam/KnowledgeBase/pull/567#discussion_r1761567299) everything else is good

**Review by el-termikael — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**